README.md

```diff
@@ -64,24 +64,41 @@ In [IBM Cloud](https://console.ng.bluemix.net/):
 
 ## 3. Building and Running
 
-If you followed the previous steps you should already be inside your local clone and ready to get started running the app from Unity.
+> Note: This has been compiled and tested using Unity 5.6.5f1 and Watson Unity SDK from the Unity asset Store (or alternately use the develop branch)
```

Comment:
Looks like the project version changed (later in this diff):

-m_EditorVersion: 5.6.1f1
+m_EditorVersion: 2017.2.0f3